### PR TITLE
Fs exercises

### PR DESCRIPTION
**Pull request by fszwedo**
exercises dashboard - I will inject it after merge as I don't want to merge master into my branch to avoid mess for code review
I already merged Konrad's branch with mine in order to be able to integrate the date picker - sorry for that (please just review the 'Exercise...' files during CR

to run please use
```npm run start:exercises```
you should have a logged in user in your localstorage with APIkey in order to test the functionality properly

EDIT: all right, now I see I probably could have merged master into my branch without any consequences, anyway please review it first and I will do it in another quick pull later for integration :)

### CODE REVIEW

**Comment by robert-czwartosz** [comment]
There is some bug in request:
```
POST https://trackapi.nutritionix.com/v2/natural/exercise
[HTTP/1.1 400 Bad Request 163ms]
```
...this communicat is in my browser console

**Comment by robert-czwartosz** [comment]
I've got solution... age shouldn't be 0

File: Src/APIConnection/Exercises.ts [hunk 1]
```diff
@@ -1,6 +1,27 @@
...
+
+async function fetchExercisesData(user: User, exercise: string, appId: string, appKey: string){
     const url = "https://trackapi.nutritionix.com/v2/natural/exercise";
-    const requestJSON = {"query": exercise}
+    const requestJSON = {
+        "query": exercise,
+        "gender": user.gender,
+        "weight_kg": user?.weights[0].weight ?? user.goalWeight, //fallback goalweight just in case the user has not provided us with anything, not sure if possible
+        "height_cm": user.height,
+        "age": getAge(user.dateOfBirth)
```
Comment (robert-czwartosz) [inline Src/APIConnection/Exercises.ts:23]: `"age": getAge(user.dateOfBirth) > 0 ? getAge(user.dateOfBirth) : 1`
should resolve issue with api request

**Review by robert-czwartosz — COMMENTED** [review]
Component looks good!
There is only one bug in request

File: Src/APIConnection/Exercises.ts (hunk 1 above)
Comment (fszwedo) [reply Src/APIConnection/Exercises.ts]: added, thanks 👍

**Review by robert-czwartosz — APPROVED** [review]

**Review by skorpiom — APPROVED** [review]

**Pull request merged**